Server/Models/users.js

```diff
@@ -1,38 +1,41 @@
 const db = require('../Database');
 
 const formatStringInputs = (str) => {
-    str = str.trim()
+    str = str.replace(/ /g, ' ')
```

Comment:
i can't find it online what does global regex empty space and replace with empty space char do?

Reply:
trim will remove spaces at the beginning and the end and ignores the rest
ex: '    this     is    an example        ` will be converted to 'this     is    an example` with trim
replace //g is a global replace .... this is a bug lol
thank you for catching that, basically I dind't do anything

Reply:
just fixed it